packages/driver/src/cy/mouse.ts

```diff
@@ -532,7 +545,7 @@ export const create = (state, keyboard, focused, Cypress) => {
 
       const mouseDownPhase = mouse.down(fromElViewport, forceEl, pointerEvtOptionsExtend, mouseEvtOptionsExtend)
 
-      const skipMouseupEvent = mouseDownPhase.events.pointerdown.skipped || mouseDownPhase.events.pointerdown.preventedDefault
+      const skipMouseupEvent = mouseDownPhase.events.pointerdown.preventedDefault
```

Comment:
By definition, `mouseDownPhase.events.pointerdown.skipped` is always `undefined`.